packages/runtime/runtime-utils/src/remoteFluidObjectHandle.ts

```diff
@@ -54,6 +54,7 @@ export class RemoteFluidObjectHandle extends FluidHandleBase<FluidObject> {
 			};
 			this.objectP = this.routeContext.resolveHandle(request).then<FluidObject>((response) => {
 				if (response.mimeType === "fluid/object") {
+					// Type assertion is safe due to mimeType check ensuring value is a FluidObject
```

Comment:
A bit clearer; checking the mime type on its own is not really a guarantee.
```suggestion
					// Responses with mimeType == "fluid/object" are produced by Fluid, and thus they can reasonably be expected to always be a FluidObject
```